src/app/views/config.coffee

```diff
@@ -48,7 +48,7 @@ module.exports = class ConfigView extends BaseView
     formatDate: (date) ->
         log.debug "formatDate #{date}"
 
-        return t 'never' unless date or date is ''
+        return t 'never' unless date or date isnt ''
```

Comment:
I think it should be an `and`, not an  `or` in the conditional. But you should write it with an `if`, not an `unless` to make it easier to reason about.